yb-voyager/src/dbzm/config.go

```diff
@@ -55,8 +55,9 @@ type Config struct {
 	SSLKeyStorePassword   string
 	SSLTrustStore         string
 	SSLTrustStorePassword string
-
-	SnapshotMode string
+	YBStreamID            string
+	YBServers             string
```

Comment:
If you really want this variable to represent master nodes only then choose the name to reflect it e.g. YBMasterNodes